Pod/Classes/SAConfettiView.swift

```diff
@@ -12,11 +12,11 @@ import QuartzCore
 public class SAConfettiView: UIView {
 
     public enum ConfettiType {
-        case confetti
-        case triangle
-        case star
-        case diamond
-        case image(UIImage)
+        case Confetti
```

Comment:
I think the latest convention is lowercase enums. https://github.com/raywenderlich/swift-style-guide#constants

Reply:
Yes you're right, my mistake